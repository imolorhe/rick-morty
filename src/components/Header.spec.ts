import Header from './Header.vue';
import { shallowMount } from '@vue/test-utils';

describe('Header', () => {
  it('should render', () => {
    const comp = shallowMount(Header, {
      stubs: {
        RouterLink: true,
      },
    });

    expect(comp.html()).toMatchSnapshot();
  });
});
