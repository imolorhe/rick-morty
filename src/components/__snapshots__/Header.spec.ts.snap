// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<nav class="flex items-center justify-between flex-wrap bg-purple-900 p-6">
  <div class="flex items-center flex-shrink-0 text-white mr-6"><img svg-inline="" src="../assets/rick.svg" alt="rick" class="logo fill-current w-10 h-10 mr-4"> <span class="font-semibold text-xl tracking-tight">
      Rick's Place
    </span></div>
  <div class="block lg:hidden"><button class="flex items-center px-3 py-2 border rounded text-purple-200 border-purple-400 hover:text-white hover:border-white"><svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" class="fill-current h-3 w-3">
        <title>Menu</title>
        <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"></path>
      </svg></button></div>
  <div class="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
    <div class="text-md lg:flex-grow">
      <routerlink-stub to="/" class="block mt-4 lg:inline-block lg:mt-0 text-purple-200 rounded-full lg:hover:bg-purple-800 lg:px-3 py-1 hover:text-white mr-4 transition duration-300">
        Home
      </routerlink-stub>
      <routerlink-stub to="/about" class="block mt-4 lg:inline-block lg:mt-0 text-purple-200 rounded-full lg:hover:bg-purple-800 lg:px-3 py-1 hover:text-white mr-4 transition duration-300">
        About
      </routerlink-stub>
      <routerlink-stub to="/characters" class="block mt-4 lg:inline-block lg:mt-0 text-purple-200 rounded-full lg:hover:bg-purple-800 lg:px-3 py-1 hover:text-white mr-4 transition duration-300">
        Characters
      </routerlink-stub>
    </div>
  </div>
</nav>
`;
